admin/app/components/solidus_admin/sidebar/account_nav/component.js

```diff
@@ -0,0 +1,13 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["links"]
+
+  // Toggle the visibility of the links
+  toggleLinks(event) {
+    event.target.classList.toggle(
+      event.target.dataset.activeClass
```

Comment:
Storing classes into attributes is so popular that Stimulus added a configuration for it: https://stimulus.hotwired.dev/reference/css-classes

You think we can switch to it?

Reply:
Cool! Done! Thanks for the tip.